cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -36,19 +36,40 @@ Feature: Users can authneticate with OIDC authenticator
       member: !user alice
 
     - !user alice@conjur.net
```

Comment:
@jtuttle I'd prefer not to have this in the background policy but to add a new policy in the `A valid id token with email as id-token-user-property` scenario. This, however, doesn't work. do you know why?

You can reproduce by removing this line and adding this to the beginning of the scenario:
```
Given a policy:
    """
   - !user alice@conjur.net
    """
```